lib/moon/components/deprecated/single_select.ex

```diff
@@ -104,7 +104,7 @@ defmodule Moon.Components.Select.SingleSelect.Value.SelectedValue do
   end
 end
 
-defmodule Moon.Components.Select.SingleSelect.Value do
```

Comment:
Please only 1 component. No deceparated.

Each application owner must upgrade his SingleSelects according to Moon API when upgrading Moon platform. 

Deceparated components cause 2 version of same component to be maintained, with confusion from both Moon development team and component users side.

Reply:
Also deceperated TextInput should be removed.